src/lib/utils.ts

```diff
@@ -1,3 +1,17 @@
 export const truncateMinaPublicKey = (key: string, front = 10, back = 10) => {
   return key.slice(0, front) + '...' + key.slice(key.length - back);
+}
+
+export const unitImage = (unitName: string): string => {
```

Comment:
this is one idea.. kind of brittle, but gets us off the ground.  If you want to take a totally diffferent approach to this, that's fine with me